packages/core/src/components/toast/toaster.tsx

```diff
@@ -268,5 +272,12 @@ export class Toaster extends AbstractPureComponent2<IToasterProps, IToasterState
     };
 }
 
-export const OverlayToaster = Toaster;
-export type OverlayToasterProps = IToasterProps;
+/** @deprecated use the new, more specific component name `OverlayToaster` instead (forwards-compatible with v5) */
+export const Toaster = OverlayToaster;
+/** @deprecated use the new, more specific type `ToasterInstance` instead (forwards-compatible with v5) */
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+export type Toaster = ToasterInstance;
```

Comment:
FYI, this is a breaking change, e.g. https://github.com/pybricks/pybricks-code/actions/runs/4989185014/jobs/8932767904?pr=1741#step:5:6

Reply:
ah, sorry about that, thanks for flagging. I suppose I could fix this by changing it to:

```suggestion
// eslint-disable-next-line @typescript-eslint/no-redeclare
export type Toaster = typeof OverlayToaster;
```

Reply:
Opened a PR: https://github.com/palantir/blueprint/pull/6165

Reply:
It should probably just be deleted until v5 to avoid complications.

Reply:
I don't think removing this line will be backwards-compatible.

Before, `Toaster` was a class, so it could be referenced as a type or a value.

After this change, it's re-exported as a value with `const Toaster = OverlayToaster`. But that will break existing usage of `Toaster` as a type (you could work around this with `typeof Toaster`).

To restore `Toaster` as a _type_ in the public API, I think we need an `export type Toaster = ...` statement.

Reply:
Ah, makes sense.